src/VisualStudio/Core/Impl/CodeModel/MethodXml/AbstractMethodXmlBuilder.cs

```diff
@@ -226,6 +228,19 @@ private AttributeInfo RankAttribute(int rank)
             return new AttributeInfo(RankAttributeName, rank.ToString());
         }
 
+        private AttributeInfo SpecialCastKindAttribute(SpecialCastKind? specialCastKind = null)
+        {
+            switch (specialCastKind)
+            {
+                case SpecialCastKind.DirectCast:
+                    return new AttributeInfo(DirectCastAttributeName, "yes");
```

Comment:
Correct. That's what WinForms expects from IMethodXml, and we use "yes"/"no" for other attributes as well (like "implicit"). FWIW, I created the unit tests by running IMethodXml in VS 2013 and capturing the output.